stdlib/LinearAlgebra/src/symmetriceigen.jl

```diff
@@ -195,3 +225,33 @@ function eigvals!(A::Hermitian{T,S}, B::Hermitian{T,S}; sortby::Union{Function,N
     return vals
 end
 eigvecs(A::HermOrSym) = eigvecs(eigen(A))
+
+# Note: No specilized LAPACK routines for Matrix+Symmetric and Symmetric+Matrix exist. Hence, the calls are forwarded to conventional Matrix eigvals!
+function eigvals!(A::StridedMatrix{T}, B::Union{RealHermSymComplexHerm{T},Diagonal{T}}; sortby::Union{Function,Nothing}=nothing) where {T<:Number}
+    return eigvals!(A, Matrix{T}(B); sortby)
+end
+
+function eigvals!(A::RealHermSymComplexHerm{T,<:StridedMatrix}, B::AbstractMatrix{T}; sortby::Union{Function,Nothing}=nothing) where {T<:Number}
+    return eigvals!(Matrix{T}(A), B; sortby)
+end
+
+function eigvals!(A::AbstractMatrix{T}, C::Cholesky{T, <:AbstractMatrix}; sortby::Union{Function,Nothing}=nothing) where {T<:Number}
+    return _choleigvals!(A, C.U; sortby)
+end
+function eigvals(A::AbstractMatrix{T}, C::Cholesky{T, <:AbstractMatrix}; sortby::Union{Function,Nothing}=nothing) where {T<:Number}
+    eigvals!(eigencopy_oftype(A, T), copy(C); sortby)
```

Comment:
Is the copy of `C` really needed?

Reply:
[ACCEPTED] Fixed in https://github.com/aravindh-krishnamoorthy/julia/commit/9c24fbdc0b94cb180709b5f45f25e093e9d460af